### PR TITLE
Added ServicePulse section for Real-time Monitoring

### PR DESCRIPTION
**Pull request by ramonsmits**
Added section Real-time Monitoring, showing the announcement video and referencing the tutorial and metric definitions.

Signed-off-by: Ramon Smits <ramon.smits@gmail.com>

### CODE REVIEW

File: servicepulse/index.md [hunk 1]
```diff
@@ -34,6 +34,18 @@ In order to monitor endpoints health and activity it is necessary to [configure
...
+## Real-time Monitoring
+
+The Monitoring view shows an overview of all logical endpoints in a system showing various metrics. Navigate to the details by clicking on an endpoint name. This shows the same metrics but then split by message type or per endpoint instance.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/X8oQjKnKpBM?rel=0" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
+
+**Learn more:**
+
+ * [Explanation of metrics](/nservicebus/operations/metrics/)
+ * [Monitoring tutorial](/tutorials/monitoring-setup/)
```
Comment (mikeminutillo) [inline servicepulse/index.md]: Is this the tutorial or should this say something like "Set up monitoring"

**Comment by ramonsmits** [comment]
@SzymonPobiega Are you fine with the HTML on the page? I'm not sure if that is allowed. If so, then it I think you can merge it.

**Comment by SzymonPobiega** [comment]
@ramonsmits I am not sure either. Let's merge and see.

**Comment by SzymonPobiega** [comment]
it looks OK.

**Pull request merged**